src/components/Sidebar.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { Logo1 } from "../assets";
+import { MdOutlineClear, MdOutlineDashboard } from "react-icons/md";
+
+import { HiOutlineUsers } from "react-icons/hi";
+import { BiMoneyWithdraw } from "react-icons/bi";
+
+import { RiAccountPinCircleFill } from "react-icons/ri";
+import { useNavigate } from "react-router-dom";
+
+interface MenuItem {
+  name: string;
+  icon: React.ReactElement;
+  path: string;
+}
+
+interface SidebarProps {
+  active: boolean;
+  setActive: (active: boolean) => void;
+}
+
```

Comment:
can you move this to the interface folder

Reply:
Okay  sure